vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb

```diff
@@ -0,0 +1,25 @@
+module SecureRooms
+
+  class OccupanciesController < ApplicationController
+
+    layout "plain"
+
+    before_action :load_occupancies
+
+    def index
+      @refresh_url = refresh_facility_secure_room_occupancies_url(current_facility, @secure_room.dashboard_token)
```

Comment:
hmm I would have to move the javascript either into the layout or into assets somehow... I'm not sure which setup I prefer